### DISCUSSION TITLE
GitHub Pages and Google index

### DISCUSSION THREAD

**Discussion by nicola-attico** [community/community#22385] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,<br>
I created a <a href="https://ethnow.github.io/" rel="noopener nofollow ugc">site</a> with GitHub pages but I’m not able to have it indexed by google.<br>
The Google console shows that the site is good to go but not indexed for many weeks (I imported the google verification file within the site repo)<br>
Seo tags seem to be there too.</p>
<p>Any suggestion to make that work?</p>
<p>Thanks</p>
<p>Nicola</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/nicola-attico">@nicola-attico</a>, welcome to the GitHub Support Community! The only thing you can really do is register your site in Google’s webmaster tools. This can take time to finish though so you may just need to wait for it to finally be indexed:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2439407/181070693-fae69eaa-097c-408f-969f-24f9220908f4.png" class="site-icon" width="32" height="32">
      <a href="https://developers.google.com/search/docs/advanced/crawling/ask-google-to-recrawl" target="_blank" rel="noopener">Google Developers</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/388;"><img src="https://user-images.githubusercontent.com/2439407/181070694-dafa8e6a-df0f-49bf-8c67-6b41e67ed030.png" class="thumbnail" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9cb039778a9c14fcd36c8f4662b5e0e315700eb2_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9cb039778a9c14fcd36c8f4662b5e0e315700eb2_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/c/9cb039778a9c14fcd36c8f4662b5e0e315700eb2.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9cb039778a9c14fcd36c8f4662b5e0e315700eb2_2_10x10.png"></div>

<h3><a href="https://developers.google.com/search/docs/advanced/crawling/ask-google-to-recrawl" target="_blank" rel="noopener">Ask Google to recrawl your URLs  |  Google Search Central</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
